vars/waitForHelixRuns.groovy

```diff
@@ -25,15 +25,25 @@ def call (def helixRunsBlob, String prStatusPrefix) {
     def passed = true
 
     for (int i = 0; i < helixRunsBlob.size(); i++) {
-        def currentRun = helixRunsBlob[i];
-        def queueId = currentRun['QueueId']
-        def correlationId = currentRun['CorrelationId']
-        def statusUrl = "https://helix.dot.net/api/2017-04-14/jobs/${correlationId}/details"
-        def statusResponse = httpRequest statusUrl
-        assert statusResponse != null
-        assert statusResponse.content != null
-        def statusContent = (new JsonSlurperClassic()).parseText(statusResponse.content)
-        def mcResultsUrl = "https://mc.dot.net/#/user/${getEncodedUrl(statusContent.Creator)}/${getEncodedUrl(statusContent.Source)}/${getEncodedUrl(statusContent.Type)}/${getEncodedUrl(statusContent.Build)}"
+        try {
+            def currentRun = helixRunsBlob[i];
+            def queueId = currentRun['QueueId']
+            def correlationId = currentRun['CorrelationId']
+            def statusUrl = "https://helix.dot.net/api/2017-04-14/jobs/${correlationId}/details"
+            println('Sending request to ' + statusUrl)
```

Comment:
I thought it would be better to track explicit user created echos but I guess I can remove this.